.github/CODEOWNERS

```diff
@@ -308,1068 +308,986 @@ app/models/veteran_onboarding.rb @department-of-veterans-affairs/backend-review-
+config/initializers/rgeo.rb @department-of-veterans-affairs/backend-review-group
+config/initializers/rswag-ui.rb @department-of-veterans-affairs/backend-review-group
+config/initializers/rubyzip.rb @department-of-veterans-affairs/backend-review-group
 config/initializers/saml.rb @department-of-veterans-affairs/octo-identity
-config/initializers/sentry.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-config/initializers/session_store.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-config/initializers/sidekiq.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+config/initializers/sentry.rb @department-of-veterans-affairs/backend-review-group
+config/initializers/session_store.rb @department-of-veterans-affairs/backend-review-group
+config/initializers/sidekiq.rb @department-of-veterans-affairs/backend-review-group
 config/initializers/sign_in_service.rb @department-of-veterans-affairs/octo-identity
-config/initializers/staccato.rb @department-of-veterans-affairs/vfs-10-10 @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-config/initializers/statsd_instrument_monkeypatch.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-config/initializers/statsd.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-config/initializers/strong_migrations.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-config/initializers/vbms.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-config/initializers/veteran_facing_services_notification_callbacks.rb @department-of-veterans-affairs/benefits-lifestage @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-config/initializers/warden_github.rb @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
-config/locales @department-of-veterans-affairs/va-api-engineers @department-of-veterans-affairs/backend-review-group
+config/initializers/staccato.rb @department-of-veterans-affairs/vfs-10-10 @department-of-veterans-affairs/backend-review-group
+config/initializers/statsd_instrument_monkeypatch.rb @department-of-veterans-affairs/backend-review-group
+config/initializers/statsd.rb @department-of-veterans-affairs/backend-review-group
+config/initializers/strong_migrations.rb @department-of-veterans-affairs/backend-review-group
+config/initializers/vbms.rb @department-of-veterans-affairs/backend-review-group
+config/initializers/veteran_facing_services_notification_callbacks.rb @department-of-veterans-affairs/pension-and-burials @department-of-veterans-affairs/backend-review-group
```

Comment:
What's the reason for `benefits-lifestage` being replaced with `pension-and-burials`?

Reply:
Good catch; TY!